hypatia/query/__init__.py

```diff
@@ -786,7 +786,11 @@ def process_Str(self, node, children):
         return node.s
 
     def process_Constant(self, node, children):
-        return node.s
+        """
```

Comment:
I would update default coverage to run under Python 3.8 instead, so that no further changes are required here, we don't have a requirement to run coverage under the lower common denominator of Python.

Reply:
I'm planning to bump the minimum required Python version to 3.8 (see #17, which I will rebase after merging this PR).